plugins/package-managers/bazel/src/main/kotlin/CompositeBazelModuleRegistryService.kt

```diff
@@ -72,6 +72,10 @@ internal class CompositeBazelModuleRegistryService(
         }
     }
 
+    // The [CompositeBazelModuleRegistryService] does not have a single URL nor is used by the
+    // [MultiBazelModuleRegistryService].
+    override fun url(): String = "Composite"
```

Comment:
Should `url` changed to be a list and return all URLs from the underlying registries?